### DISCUSSION TITLE
Rain in inch units

### DISCUSSION THREAD

**Discussion by tguozden** [bellrichm/WeeWX-MQTTSubscribe#222] [General]
When reading rain values in inch units weewx gets the value times 10. Does not happen when working in mm.

Sorry I did the comment with another user, I cannot put it down.

**Comment by UNRNMETEO** [comment]
I am using METRICWX for database units, in which rain is expressed in centimeters i think.

**Reply by bellrichm** [reply]
METRICWX has rain in mm, see http://www.weewx.com/docs/5.1/reference/units/?h=metricwx
If you set debug = 1, restart WeeWX, let in run through at least one archive interval and post the log; I can take a look.
In addition to the log, please post your configuration for MQTTSubscribe. 
Make sure to remove any sensitive information.

**Reply by UNRNMETEO** [reply]
My fault, was a 60' cumulative variable and I didn't set contains_total to true.. Sorry!

Thanks anyway for your quick response and for the very useful repos you share and support.

**Reply by bellrichm** [reply]
No problem. Thanks for letting me know!
